test/surface_test.py

```diff
@@ -284,8 +310,28 @@ def test_subsurface_rle2(self):
         s2 = s1.subsurface(sub_rect)
         s2.fill(color2)
         s0.blit(s2, (0, 0))
-        self.assertTrue(s1.get_flags() & pygame.RLEACCEL)
-        self.assertTrue(not s2.get_flags() & pygame.RLEACCEL)
+        self.assertTrue(s1.get_flags() & pygame.RLEACCELOK)
+        self.assertTrue(not s2.get_flags() & pygame.RLEACCELOK)
+
+    def test_solarwolf_rle_usage(self):
+        """ Test for error/crash when calling set_colorkey() followed
+            by convert twice in succession. Code originally taken
+            from solarwolf. """
+        def optimize(img):
+            clear = img.get_colorkey()
+            img.set_colorkey(clear, RLEACCEL)
+            return img.convert()
+
+        pygame.display.init()
+        try:
+            pygame.display.set_mode((640, 480))
+            image = pygame.image.load(example_path(os.path.join("data",
+                                                                "alien1.png")))
+            image = optimize(image)
+            image = optimize(image)
+            self.assertTrue(isinstance(image, pygame.Surface))
```

Comment:
I think this should also assert about the color key, and the RLEACCEL state.

Reply:
done.